2021-11-05 14:20:38,582 INFO    MainThread:3906 [wandb_setup.py:_flush():69] setting env: {'run_group': 'dev', 'project': 'exercise_14'}
2021-11-05 14:20:38,582 INFO    MainThread:3906 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-05 14:20:38,582 INFO    MainThread:3906 [wandb_init.py:_log_setup():334] Logging user logs to /home/default/github/genre_classification/starter/preprocess/wandb/run-20211105_142038-2e972f36/logs/debug.log
2021-11-05 14:20:38,582 INFO    MainThread:3906 [wandb_init.py:_log_setup():335] Logging internal logs to /home/default/github/genre_classification/starter/preprocess/wandb/run-20211105_142038-2e972f36/logs/debug-internal.log
2021-11-05 14:20:38,584 INFO    MainThread:3906 [wandb_init.py:init():367] calling init triggers
2021-11-05 14:20:38,584 INFO    MainThread:3906 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-11-05 14:20:38,584 INFO    MainThread:3906 [wandb_init.py:init():416] starting backend
2021-11-05 14:20:38,584 INFO    MainThread:3906 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-05 14:20:38,616 INFO    MainThread:3906 [backend.py:ensure_launched():123] starting backend process...
2021-11-05 14:20:38,630 INFO    MainThread:3906 [backend.py:ensure_launched():127] started backend process with pid: 3952
2021-11-05 14:20:38,632 INFO    MainThread:3906 [wandb_init.py:init():421] backend started and connected
2021-11-05 14:20:38,633 INFO    MainThread:3906 [wandb_init.py:init():461] updated telemetry
2021-11-05 14:20:38,634 INFO    MainThread:3906 [wandb_init.py:init():480] communicating current version
2021-11-05 14:20:41,748 INFO    MainThread:3906 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-05 14:20:41,748 INFO    MainThread:3906 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-11-05 14:20:42,649 INFO    MainThread:3906 [wandb_init.py:init():518] starting run threads in backend
2021-11-05 14:20:44,008 INFO    MainThread:3906 [wandb_run.py:_console_start():1476] atexit reg
2021-11-05 14:20:44,010 INFO    MainThread:3906 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-11-05 14:20:44,010 INFO    MainThread:3906 [wandb_run.py:_redirect():1344] Redirecting console.
2021-11-05 14:20:44,010 INFO    MainThread:3906 [redirect.py:install():213] install start
2021-11-05 14:20:44,011 INFO    MainThread:3906 [redirect.py:install():228] install stop
2021-11-05 14:20:44,011 INFO    MainThread:3906 [redirect.py:install():213] install start
2021-11-05 14:20:44,011 INFO    MainThread:3906 [redirect.py:install():228] install stop
2021-11-05 14:20:44,011 INFO    MainThread:3906 [wandb_run.py:_redirect():1390] Redirects installed.
2021-11-05 14:20:44,011 INFO    MainThread:3906 [wandb_init.py:init():542] run started, returning control to user process
2021-11-05 14:20:46,328 INFO    MainThread:3906 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-11-05 14:20:46,328 INFO    MainThread:3906 [wandb_run.py:_restore():1418] restore
2021-11-05 14:20:46,328 INFO    MainThread:3906 [redirect.py:uninstall():232] uninstall start
2021-11-05 14:20:46,328 INFO    MainThread:3906 [redirect.py:_stop():287] _stop: stdout
2021-11-05 14:20:46,328 INFO    MainThread:3906 [redirect.py:_stop():293] _stop closed: stdout
2021-11-05 14:20:46,328 INFO    stdout    :3906 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-11-05 14:20:46,328 INFO    stdout    :3906 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-11-05 14:20:46,328 INFO    MainThread:3906 [redirect.py:_stop():299] _stop joined: stdout
2021-11-05 14:20:46,328 INFO    MainThread:3906 [redirect.py:_stop():301] _stop rd closed: stdout
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:uninstall():236] uninstall done
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:uninstall():232] uninstall start
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:_stop():287] _stop: stderr
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:_stop():293] _stop closed: stderr
2021-11-05 14:20:46,329 INFO    stderr    :3906 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-11-05 14:20:46,329 INFO    stderr    :3906 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:_stop():299] _stop joined: stderr
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:_stop():301] _stop rd closed: stderr
2021-11-05 14:20:46,329 INFO    MainThread:3906 [redirect.py:uninstall():236] uninstall done
2021-11-05 14:20:47,083 INFO    MainThread:3906 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1069
  total_bytes: 1069
}

2021-11-05 14:20:49,085 INFO    MainThread:3906 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1949
  total_bytes: 1949
}

2021-11-05 14:20:50,155 INFO    MainThread:3906 [wandb_run.py:_show_files():1790] logging synced files
